src/components/Deck.tsx

```diff
@@ -44,14 +44,14 @@ const Deck = ({
         {isHighlighted && (
           <mesh>
             <RoundedRectangleGeometry args={[cardWidth + 8, cardHeight + 8, 1.5, 0.01]} />
```

Comment:
I think we have to update the border radius of the decks as well, they look different now